resources/core/templates/tests/rbac-external-solution.yaml

```diff
@@ -55,6 +55,15 @@ rules:
   - apiGroups: ["gateway.kyma-project.io"]
     resources: ["apis"]
     verbs: ["create", "delete"]
+  - apiGroups: ["messaging.knative.dev"]
```

Comment:
Why are the create permissions required for channels and brokers? Also delete?

Reply:
we would still support the old test, there  subscriptions are created from the test which are now part of messaging api group, To unify all the messaging group. i gave default access to the test service account

Reply:
As discussed: we reduced the RBAC permissions and removed write access there possible